eras/conway/impl/src/Cardano/Ledger/Conway/Era.hs

```diff
@@ -39,6 +40,7 @@ instance Crypto c => Era (ConwayEra c) where
   type PreviousEra (ConwayEra c) = BabbageEra c
   type EraCrypto (ConwayEra c) = c
   type ProtVerLow (ConwayEra c) = 9
+  -- type ProtVerHigh (ConwayEra c) = 10
```

Comment:
I'm not sure if we want to have an upper bound to the protocol before the next era is out, @lehins probably knows more about this.

Reply:
Protocol version 10 is not necessarily Conway. So this would be wrong